sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -56,6 +56,7 @@ import {
   OFFERING_BULK_UPLOAD_MAX_UPLOAD_PARTS,
   uploadLimits,
 } from "../../utilities";
+import { ParseEnumQueryPipe } from "../../utilities/class-validation";
```

Comment:
`ParseEnumQueryPipe` it is not a `class-validator`, IMO, and I believe that belongs to the controllers' scope only, doesn't it?
I would say that it can be created inside the `route-controllers/utils`, thoughts?

Reply:
All good.